src/App.js

```diff
@@ -44,9 +44,18 @@ import ModalAlert from './components/ModalAlert';
 import SoftwareWalletWarningMessage from './components/SoftwareWalletWarningMessage';
 import AddressList from './screens/AddressList';
 
-hathorLib.network.setNetwork('mainnet');
 hathorLib.storage.setStore(new HybridStore());
 
+// Default if data on storage is null is mainnet
+// because we currently support only mainnet
+const currentNetwork = hathorLib.storage.getItem('wallet:network') || 'mainnet';
+// We always set the network in the version.js utils file when we receive the /version/ from the full node
+// however when we first start the app we also need to set it because this /version/ request is async
+// and we might start loading the transactions before we receive it.
```

Comment:
I think we should refactor it before merging the PR. Why wasn't it a problem in the current version of the wallet?

Reply:
It was not a problem because we weren't changing the network. I've just done this refactor and now this code is not needed anymore. Can you check again please?

Reply:
Great! Thanks!